<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewPlayerWindow</class>
 <widget class="QWidget" name="PreviewPlayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="VlcWidgetVideo" name="videoAreaWidget" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="VlcWidgetVolumeSlider" name="volumeWidget" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButtonPlayPause">
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButtonOpenFile">
     <property name="text">
      <string>Open Video Stream</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="VlcWidgetSeek" name="seekWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VlcWidgetVideo</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetVideo.h</header>
   <container>1</container>
   <slots>
    <signal>fullscreen()</signal>
    <slot>toggleFullscreen()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>VlcWidgetVolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetVolumeSlider.h</header>
   <container>1</container>
   <slots>
    <slot>mute()</slot>
    <slot>volumeUp()</slot>
    <slot>volumeDown()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>VlcWidgetSeek</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetSeek.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
